index.php

```diff
@@ -89,9 +89,23 @@ public function __construct(string $baseDir) {
 		if (!file_exists($configFileName)) {
 			throw new \Exception('Could not find config.php. Is this file in the "updater" subfolder of Nextcloud?');
 		}
+		$filePointer = @fopen($configFileName, 'r');
+		if ($filePointer === false) {
+			throw new \Exception('Could not open config.php.');
```

Comment:
```suggestion
			throw new \Exception('Could not open config.php ' . $configFileName);
```

Reply:
I'm a fan of including the `$configFileName` in the output (there are other spots too) since:

- full paths being on-hand helps make troubleshooting easier
- there may eventually be support for multiple config files at the Updater level (like there is already at Server)